src/main/kotlin/string_calculator/Calculation.kt

```diff
@@ -0,0 +1,23 @@
+package string_calculator
+
+import java.util.function.BiFunction
+
+enum class Calculation(val operator: Char, val calculation: BiFunction<Int, Int, Int>) {
```

Comment:
enum 클래스 활용 좋습니다 👍 
BiFunction을 활용해주셨는데요, Kotlin lambda를 적용하여 다음과 같이 표현할 수도 있어요.
```suggestion
enum class Calculation(val operator: Char, val calculation: (Int, Int) -> Int) {
```